src/libraries/System.Security.Cryptography/ref/System.Security.Cryptography.cs

```diff
@@ -3139,6 +3140,15 @@ public sealed partial class X509Store : System.IDisposable
         public void Remove(System.Security.Cryptography.X509Certificates.X509Certificate2 certificate) { }
         public void RemoveRange(System.Security.Cryptography.X509Certificates.X509Certificate2Collection certificates) { }
     }
+    public partial class X509SubjectAlternativeNameExtension : System.Security.Cryptography.X509Certificates.X509Extension
```

Comment:
The [Approved API](https://github.com/dotnet/runtime/issues/22699#issuecomment-1184787969) has this class marked as `sealed`.